content/guides/getting-started/component-framework-configuration.md

```diff
@@ -29,58 +45,28 @@ Next, follow the instructions for your framework.
 
 ## React
 
-### Create React App (CRA)
-
-For [Create React App](https://create-react-app.dev/), you will need to install
-the Cypress React Adapter as well as the Cypress Webpack Dev Server as dev
-dependencies:
-
-<npm-or-yarn>
-<template #npm>
-
-```sh
-npm install --save-dev @cypress/react @cypress/webpack-dev-server
-```
-
-</template>
-<template #yarn>
-
-```sh
-yarn add -D @cypress/react @cypress/webpack-dev-server
-```
-
-</template>
-</npm-or-yarn>
+For React apps, we have built-in support for Create React App, Next.js, Vite,
+and a custom webpack config.
 
-<Alert type="info">
-
-<strong class="alert-header">Note</strong>
-
-If you are using Create React App v4, additionally install
-`html-webpack-plugin@4`, which works with the version of Webpack that comes with
-CRAv4.
-
-</Alert>
-
-Next, you will need to configure component testing.
-
-A dev server is needed to launch component testing. Cypress provides a dev
-server that works with CRA out of the box. Import 'devServer' from
-`@cypress/react/plugins/react-scripts`, then pass in the `devServer` to the
-component configuration in the cypress config file.
+### Create React App (CRA)
 
-Copy the following into your cypress config file:
+To configure component testing for a React app that uses
+[Create React App](https://create-react-app.dev/), you will need to configure a
```

Comment:
Can you make sure that throughout this guide, `framework` and `bundler` are formatted as code (using backticks) and link out to the `devServer` section of the config guide?

Note that that section is currently called `devServer / devServerConfig`, which will likely change to `devServer` as part of these docs updates.

Reply:
I think this guide is the more appropriate place for devServer configuration info. The config ref guide links back to here.